docs/_includes/variation-content.html

```diff
@@ -40,62 +40,78 @@ <h2 class="o-variation_group_title">{{ variation_group.variation_group_name }}</
         {% for variation in variation_group.variations %}
         <div class="m-variation">
 
-            {% if variation.variation_name %}
+            {% assign variation_name = variation.variation_name | strip %}
+            {% assign variation_description = variation.variation_description | strip %}
+            {% assign variation_code_snippet = variation.variation_code_snippet %}
+
+            {% unless variation_name == '' %}
             <div class="m-variation_name">
-                <h3>{{ variation.variation_name }}</h3>
+                <h3>{{ variation_name }}</h3>
             </div>
-            {% endif %}
+            {% endunless %}
 
-            {% if variation.variation_description %}
+            {% unless variation_description == '' %}
             <div class="m-variation_description">
-                {{ variation.variation_description | markdownify }}
+                {{ variation_description | markdownify }}
```

Comment:
Why not apply all the filters when the variable is created on L:44? Will it ever be used without markdownify?

Reply:
Good call. 👍